### DISCUSSION TITLE
Error on Installing Homebrew from command line

### DISCUSSION THREAD

**Discussion by CaseyJNova** [Homebrew/discussions#378] [Getting started]

**Comment by Rylan12** [comment]
As the output says, run:

```
git -C /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core fetch --unshallow
```

**Reply by CaseyJNova** [reply]
Thank you and right you are!

I think the issue was that I had to type that in, hit enter, then wait about 3 minutes for it to start.

Perhaps this is totally normal but it was throwing me for a loop.

Works great now and I thank you for the help!
